README.md

```diff
@@ -97,7 +97,7 @@ packages that improve performance (if you know of others please let us know).
 Package | Version | Improvements
 --------|---------|-------------
 PHP     | 7.4+    | PHP 7.4 includes a number of performance improvements that increase perf by up to 30%
-Python3 |         | Python3 packages in Ubuntu, Amazon Linux 2 and RHEL were not compiled with link time optimizations (LTO). We have seen up to 30% better performance with LTO.
+Python3 |         | Python3 packages in Ubuntu, Amazon Linux 2 and RHEL were not compiled with link time optimizations (LTO). We have seen up to 30% better performance with LTO.  Also see [improvements to Python runtime](python.md).
```

Comment:
This has been addressed now. (lack of LTO). We can remove this right?

Reply:
I removed the LTO text and I only left the link to the python page.